src/Config.php

```diff
@@ -909,6 +909,7 @@ protected function getDefaults()
                 'dbname'         => 'bolt',
                 'prefix'         => 'bolt_',
                 'charset'        => 'utf8',
```

Comment:
> is it ok to have a default "collate" here?

I believe it is OK. If I understand correctly, the parameter just gets ignored for other platforms.